sage/pallet/src/tests/extrinsics/buy_asset.rs

```diff
@@ -56,7 +56,11 @@ fn buy_should_work() {
 				asset_for_sale,
 				asset_price
 			));
-			assert_ok!(Sage::buy_asset(RuntimeOrigin::signed(ALICE), asset_for_sale, Some(NATIVE)));
+			assert_ok!(Sage::buy_asset(
+				RuntimeOrigin::signed(ALICE),
+				asset_for_sale,
+				MAYBE_NATIVE_PAYMENT
```

Comment:
Hmmm, I think `maybe` is misleading, as it implies that it could be none. I prefer to remove the prefix and then to rename the variable if we put none there.